gatsby/src/pages/404.tsx

```diff
@@ -1,12 +1,39 @@
 import React from "react";
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
+import { Breadcrumbs, Breadcrumb } from "@nice-digital/nds-breadcrumbs";
 
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
-			<h1>Page not found</h1>
+			<Breadcrumbs>
+				<Breadcrumb to="https://www.nice.org.uk/">NICE</Breadcrumb>
+				<Breadcrumb to="/">CKS</Breadcrumb>
+				<Breadcrumb>Page not found</Breadcrumb>
+			</Breadcrumbs>
+			<header role="banner" className="page-header">
```

Comment:
Can you just use the react component like this please?

https://github.com/nice-digital/cks-gatsby/blob/d40489b41c727d07c57a20862b861a7daf513be9/gatsby/src/templates/Topic/Topic.tsx#L43-L46